lib/event-handlers.js

```diff
@@ -55,10 +63,12 @@ module.exports = function (job, collection) {
 
       if (job.meta.id) {
         return collection.then((coll) => coll.update({_id: job.meta.id}, {$set: data}))
-          .catch(function (e) {
-            raygun.send(e);
-            log.error(e, 'failed to write TICK_COMPLETE data');
-            return;
+          .catch((e) => {
+            raygun.send(e)
+            .then(() => {
+              log.info('raygun error sent');
+              log.error(e, 'failed to write TICK_COMPLETE data');
```

Comment:
Hm.  Good point, there's no reason to put all that in the `.then`.  I'll change that